pkg/jx/cmd/delete_preview.go

```diff
@@ -98,6 +98,25 @@ func (o *DeletePreviewOptions) Run() error {
 }
 
 func (o *DeletePreviewOptions) deletePreview(name string) error {
+	jxClient, ns, err := o.JXClient()
+	if err != nil {
+		return err
+	}
+
+	environment, err := kube.GetEnvironment(jxClient, ns, name)
+	if err != nil {
+		return err
+	}
+	if environment != nil {
+		if releaseName := environment.Annotations[AnnotationReleaseName]; len(releaseName) > 0 {
```

Comment:
Maybe move this check in a function in `pkg/kube/env.go`.  I think `func IsPreview(env *v1.Environment) bool` sounds like a good name.

Reply:
there was already an `IsPreviewEnvironment` func, and I needed to get back the release name, so I created a `GetPreviewEnvironmentReleaseName(env) string` func